tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/AnalyzerTests.cs

```diff
@@ -150,10 +150,18 @@ public class AnalyzerTests
                 "UA0013",
                 new[]
                 {
-                    new ExpectedDiagnostic("UA0013", new TextSpan(143, 13)),
-                    new ExpectedDiagnostic("UA0013", new TextSpan(571, 29)),
-                    new ExpectedDiagnostic("UA0013", new TextSpan(153, 13), Language.VisualBasic),
-                    new ExpectedDiagnostic("UA0013", new TextSpan(439, 29), Language.VisualBasic),
+                    new ExpectedDiagnostic("UA0013", new TextSpan(166, 13)),
```

Comment:
why did the indexes for nearly every analyzer move?

Reply:
Previously the diagnostic's location was just the simple name of the identifier that needs updated. This PR changes the analyzer to instead mark the entire name as the location so that the code fix can work on the location it gets directly instead of having to find the fully qualified name at that point. Most of these changes are updating the expected diagnostic locations to include the namespaces for qualified type names.